modules/project_cleanup/main.tf

```diff
@@ -25,8 +30,9 @@ resource "google_service_account" "project_cleaner_function" {
 }
 
 resource "google_organization_iam_member" "project_owner" {
```

Comment:
Please just use the IAM module instead of building this list yourself: https://github.com/terraform-google-modules/terraform-google-iam/tree/master/modules/organizations_iam

Reply:
Reworked